content/english/sql_workshop/syntax_explanation/_syntax_explanation.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Explain Syntax"
+description: "Explain Syntax"
+date: 2020-07-15
+prereq: "SELECT Command, conditions (WHERE command)"
+difficulty: "Beginner"
+draft: false
+translationKey: "sql_activity_2.5"
+---
+
+<!-- Embed YouTube Video Link here when ready -->
+
+# An Email from the Galactic Federation?
+In the middle of your journey, you suddenly receive a message from the Galactic Federation. What did they send you?
+
+#
```

Comment:
I'd prefer using `<br/>` if you absolutely need to put a line break.

Reply:
Got it! Will do!